packages/core/helper-plugin/src/components/FilterListURLQuery.tsx

```diff
@@ -162,8 +186,17 @@ const AttributeTag = ({ attribute, filter, onClick, operator, value }: Attribute
   return (
     <Box padding={1}>
       <Tag onClick={handleClick} icon={<Cross />}>
-        {content}
+        <span onClick={handleContentClick}>{content}</span>
       </Tag>
+      {isVisible && (
+        <FilterPopoverURLQuery
+          displayedFilters={filtersSchema}
+          isVisible={true}
+          onToggle={handleToggle}
+          source={buttonRef}
+          initialFilter={filter}
+        />
+      )}
```

Comment:
to change the position of this dropdown you need to assign the buttonRef to a button but in your case you are passing it just to the FilterPopoverURLQuery component
Please take a look how the FilterPopoverURLQuery is used in other places for example in this file packages/core/admin/admin/src/pages/Settings/components/Filters.tsx